src/migration/1716976350324-AlertFeedSettingsFeedback.ts

```diff
@@ -0,0 +1,27 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class AlertFeedSettingsFeedback1716976350324
+  implements MigrationInterface
+{
+  name = 'AlertFeedSettingsFeedback1716976350324';
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(
+      `ALTER TABLE "alerts" ADD "lastFeedSettingsFeedback" TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT now()`,
+    );
+    await queryRunner.query(
+      `
+        UPDATE  public.alerts AS a
+        SET     "lastFeedSettingsFeedback" = u."createdAt"
+        FROM    public.user AS u
+        WHERE   u.id = a."userId"
+      `,
+    );
```

Comment:
@capJavert let me know if you want me to put this in a bin script instead.

Reply:
It is gonna update all rows in the table so it might be heavy so I would prefer it in bin so we can also potentially re-run if we have to.

Reply:
Since it's a transaction are we sure it's gonna trigger CDC? (which is my primary concern)

Reply:
We should be good since we don't track CDC changes for the `alerts` table (not anymore since sunsetting the gateway repo).